specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/CreateBillingProfile.json

```diff
@@ -0,0 +1,71 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "billingAccountName": "{billingAccountName}",
+    "parameters": {
+      "displayName": "Finance",
+      "address": {
+        "firstName": "Test",
+        "lastName": "User",
+        "addressLine1": "1 Microsoft Way",
+        "city": "Redmond",
+        "postalCode": "98052",
+        "region": "WA",
+        "country": "US"
+      },
+      "enableAzureSKUs": [
+        {
+          "skuId": "0001"
+        },
+        {
+          "skuId": "0002"
+        }
+      ],
+      "invoiceEmailOptIn": true,
+      "poNumber": "ABC12345"
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/operationResults/ed2975a8-8f77-43bb-a717-ace54326b14b?api-version=2018-07-31",
+        "Retry-After": "60",
+        "Azure-AsyncOperation": "https://management.azure.com/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/operationStatus/ed2975a8-8f77-43bb-a717-ace54326b14b?api-version=2018-07-31"
+      }
+    },
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/billingProfileId1",
+        "name": "billingProfileId1",
+        "type": "Microsoft.Billing/billingProfiles",
+        "properties": {
+          "displayName": "Finance",
+          "address": {
+            "firstName": "Test",
+            "lastName": "User",
+            "addressLine1": "1 Microsoft Way",
+            "city": "Redmond",
+            "postalCode": "98052",
+            "region": "WA",
+            "country": "US"
+          },
+          "currency": "USD",
+          "enabledAzureSKUs": [
+            {
+              "skuId": "0001",
```

Comment:
According to the schema `skuId` is read-only thus can't be specified as a parameter, see https://dev.azure.com/azure-sdk/public/_build/results?buildId=46947 for the model validation failure.